### DISCUSSION TITLE
Can I upgrade non-ha to ha?

### DISCUSSION THREAD

**Discussion by GodSpeedXI** [argoproj/argo-cd#5758] [Q&A]
I have tried to find the answer but, it's not quite clear. have anyone tried to do this? please share the idea.

**Comment by jannfis** [comment]
Hey @GodSpeedXI , this is as simple as applying the HA manifests over the non-HA manifests in your cluster, but take care to use the manifests from the same version of Argo CD.

This will leave just the non-HA version of Redis around (`argocd-redis` deployment and service resources), which you might want to delete.
